pubspec.yaml

```diff
@@ -0,0 +1,12 @@
+name: dart2_fix
+description: A tool to migrate API usage to Dart 2.
+version: 1.0.0
+
+environment:
+  sdk: '>=1.20.1 <2.0.0'
```

Comment:
Should be limited to `>=2.0.0-dev.29.0` – where the new consts were documented...

Reply:
would like this more open ended in terms of what it'll install into, so that why it won't install isn't a mystery to users